polaris-react/src/components/Badge/index.ts

```diff
@@ -1 +1,2 @@
 export * from './Badge';
+export * from './types';
```

Comment:
Do we need to consume these types outside Polaris? If yes, they will also need to be exported from https://github.com/Shopify/polaris/blob/main/polaris-react/src/index.ts.